content/integrations/postgresql.md

```diff
@@ -42,63 +40,87 @@ When it prompts for a password, enter the one used in the first command.
           - host: localhost
             port: 5432
 
-
 1.  Restart the agent.
 
-### Configuration Options
+## Configuration Options
 
-* `username` (Optional)
-* `password` (Optional)
-* `dbname` (Optional) - Name of the database you want to monitor
-* `ssl` (Optional) - Defaults to False. Whether to use SSL to connect.
-* `tags` (Optional) - Dictionary of tags
-* `relations` (Optional) - Dictionary of per-relation (table) metrics that you want to track. Each relation generates 10 metrics + 10 metrics per index.
-
-  By default all schemas are included. To track relations from specific schemas only, use the following syntax:
+* `username` (Optional) - The user account used to collect metrics, set in the [Installation section above](#installation)
+* `password` (Optional) - The password for the user account.
+* `dbname` (Optional) - The name of the database you want to monitor.
+* `ssl` (Optional) - Defaults to False. Indicates whether to use an SSL connection.
+* `use_psycopg2` (Optional) - Defaults to False. Setting this option to `True` will force the Datadog Agent to collect PostgreSQL metrics using psycopg2 instead of pg8000. Note that pyscopg2 does not support SSL connections.
+* `tags` (Optional) - A list of tags applied to all metrics collected. Tags may be simple strings or key-value pairs.
+* `relations` (Optional) - By default, all schemas are included. Add specific schemas here to collect metrics for schema relations. Each relation will generate 10 metrics and an additional 10 metrics per index. Use the following structure to declare relations:
 
       relations:
-        - relation_name: another_table
+        - relation_name: my_relation
           schemas:
-            - public
-            - prod
+            - my_schema_1
+            - my_schema_2
 
 * `collect_function_metrics` (Optional) - Collect metrics regarding PL/pgSQL functions from pg_stat_user_functions
-* `collect_count_metrics` (Optional) - Collect count metrics, default value is True for backward compatibility but they migth be slow, suggested value is False.
+* `collect_count_metrics` (Optional) - Collect count metrics. The default value is `True` for backward compatibility, but this might be slow. The recommended value is `False`.
+
+
+## Custom metrics
+
+Datadog will generate PostgreSQL metrics from custom query results. For each custom query, four components are required: `descriptors`, `metrics`, `query`, and `relation`.
```

Comment:
> Datadog will generate

What is 'Datadog' in this case? When I read that, normally I take it to mean the web app or backend.

I guess all the action of receiving and storing metrics technically does happen on the backend, but in docs for Agent checks, I normally refer to 'the Agent' as the actor. "The Agent submits", "The Agent emits", "The Agent collects", etc.